src/OrleansVSTools/OrleansVSTools/source.extension.vsixmanifest

```diff
@@ -34,4 +34,7 @@
     <Asset Type="Microsoft.VisualStudio.ProjectTemplate" d:Source="Project" d:ProjectName="VSProjectTemplateGrainInterfaceCSharp" d:TargetPath="|VSProjectTemplateGrainInterfaceCSharp;TemplateProjectOutputGroup|" Path="ProjectTemplates" d:VsixSubPath="ProjectTemplates" />
     <Asset Type="Microsoft.VisualStudio.ProjectTemplate" d:Source="Project" d:ProjectName="VSProjectSiloHostCSharp" d:TargetPath="|VSProjectSiloHostCSharp;TemplateProjectOutputGroup|" Path="ProjectTemplates" d:VsixSubPath="ProjectTemplates" />
   </Assets>
+  <Prerequisites>
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
```

Comment:
Does version [15.0,) work? Otherwise, do we know this will not work on VS 16? I have no idea how it works